zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/ZombieMonitor.scala

```diff
@@ -10,14 +10,16 @@ import cats.mtl.ApplicativeAsk
 import fs2.Stream
 import io.chrisdavenport.log4cats.StructuredLogger
 import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
-import org.broadinstitute.dsde.workbench.google2.{GKEService, GoogleDiskService, GoogleStorageService}
+import org.broadinstitute.dsde.workbench.google2.{GKEService, GoogleDiskService}
 import org.broadinstitute.dsde.workbench.model.TraceId
 
 object ZombieMonitor {
   def run[F[_]: ConcurrentEffect: Parallel](isDryRun: Boolean,
                                             ifRunAll: Boolean,
```

Comment:
Would be nice to rename  ^^^ too
```suggestion
                                            shouldRunAll: Boolean,
```